packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -128,8 +129,15 @@ export function initialFetchSpaces(sparkInstance) {
     dispatch(updateSpacesStatus({isFetching: true}));
     return dispatch(fetchSpaces(sparkInstance, {
       sinceDate: moment().subtract(`weeks`, 1).valueOf(),
-      conversationsLimit: 50
+      conversationsLimit: 10
     }))
+    .then(({spaces}) => {
+      const maxDate = _.last(spaces).lastReadableActivityDate;
+      return dispatch(fetchSpaces(sparkInstance, {
+        maxDate,
+        conversationsLimit: 50
+      }));
+    })
```

Comment:
Oh great. Should I just skip the since portion and just load 50 after the first 10?